app/javascript/shared/components/tags.jsx

```diff
@@ -290,12 +300,14 @@ class Tags extends Component {
         this.clearSelectedSearchResult();
       }
     } else if (
-      (e.keyCode < 65 || e.keyCode > 90) &&
```

Comment:
65-90 corresponds to A-Z - `#isNumberOrLetterKeyCode` checks 65-90 and also 48-57 for 0-9 input.